plutus-core/executables/Common.hs

```diff
@@ -6,47 +6,47 @@
 
 module Common where
 
-import           PlutusPrelude                            (through)
-
-import qualified PlutusCore                               as PLC
-import           PlutusCore.Check.Uniques                 as PLC (checkProgram)
-import           PlutusCore.Error                         (AsParseError, AsUniqueError, UniqueError)
-import           PlutusCore.Evaluation.Machine.ExBudget   (ExBudget (..), ExRestrictingBudget (..))
-import           PlutusCore.Evaluation.Machine.ExMemory   (ExCPU (..), ExMemory (..))
-import qualified PlutusCore.Generators                    as Gen
-import qualified PlutusCore.Generators.Interesting        as Gen
-import qualified PlutusCore.Generators.Test               as Gen
-import qualified PlutusCore.Pretty                        as PP
-import           PlutusCore.Rename                        (rename)
-import qualified PlutusCore.StdLib.Data.Bool              as StdLib
-import qualified PlutusCore.StdLib.Data.ChurchNat         as StdLib
-import qualified PlutusCore.StdLib.Data.Integer           as StdLib
-import qualified PlutusCore.StdLib.Data.Unit              as StdLib
-
-import qualified UntypedPlutusCore                        as UPLC
-import qualified UntypedPlutusCore.Check.Uniques          as UPLC (checkProgram)
-import qualified UntypedPlutusCore.Evaluation.Machine.Cek as Cek
-import qualified UntypedPlutusCore.Parser                 as UPLC (parseProgram)
-
-import           Control.DeepSeq                          (NFData, rnf)
-import           Control.Monad.Except
-import           Data.Bifunctor                           (second)
-import qualified Data.ByteString.Lazy                     as BSL
-import           Data.Foldable                            (traverse_)
-import qualified Data.HashMap.Monoidal                    as H
-import           Data.List                                (nub)
-import qualified Data.List                                as List
-import qualified Data.Text                                as T
-import           Data.Text.Encoding                       (encodeUtf8)
-import qualified Data.Text.IO                             as T
-import           Data.Traversable                         (for)
-import           Flat                                     (Flat, flat, unflat)
-import           Prettyprinter                            (Doc, pretty, (<+>))
-
-import           System.CPUTime                           (getCPUTime)
-import           System.Exit                              (exitFailure, exitSuccess)
-import           System.Mem                               (performGC)
-import           Text.Printf                              (printf)
+import PlutusPrelude (through)
+
+import PlutusCore qualified as PLC
```

Comment:
I know we voted, but I'm not convinced this is better, and this file's a good example.  I think (a) it's more difficult to see which modules are imported because what was a very clear list before is now squashed inside a big mass of text, (b) when multiple modules are imported using the same name that's not immediately obvious because the occurrences of the name aren't aligned, and (c) the explicitly imported things appear with different indentations (and are again squashed up against other stuff) so it's difficult to quickly see where something's been imported.  The response may be "Ah, but your IDE will tell you those things without having to look at the imports!", but in that case you won't be looking at the imports so do you really care what they look like?

I suppose I'll just have to get used to it though.

Reply:
I think it'll be okay, and I think the diff-friendliness will be quite nice. We can change our mind if we really hate it, but I think I'm going to err on the side of trying something new.